packages/react-components/src/components/ImageGallery.tsx

```diff
@@ -157,35 +156,23 @@ export const ImageGallery = (props: ImageGalleryProps): JSX.Element => {
   const renderBodyWithLightDismiss = (): JSX.Element => {
     return (
       <Stack className={mergeStyles(bodyContainer)} onClick={() => props.onDismiss()}>
-        <FocusTrapZone
-          onKeyDown={(e) => {
-            if (e.key === 'Escape' || e.key === 'Esc') {
```

Comment:
Does this disable dismiss when Escape is pressed?

Reply:
> Does this disable dismiss when Escape is pressed?

Removing this would not disable Escape key to dismiss. Those line was needed only to overwrite FocusTrapZone's onKeyDown behaviour.